runner-plugin-sdk/src/main/java/org/apache/apisix/plugin/runner/HttpResponse.java

```diff
@@ -194,6 +197,14 @@ private int buildStopResp(FlatBufferBuilder builder) {
         Stop.startStop(builder);
         if (!Objects.isNull(statusCode)) {
             Stop.addStatus(builder, statusCode);
+        } else {
+            /**
+             * Avoid APISIX using 0 as the default HTTP Status Code
+             * {@link org.apache.apisix.plugin.runner.HttpResponse#setStatusCode(int statusCode)}
+             * @see https://github.com/apache/apisix-java-plugin-runner/issues/55
+            */
+            Stop.addStatus(builder, 500);
```

Comment:
Is 500 a suitable value for the default one?

Reply:
Not very suitable and can cause some ambiguity, do you have any suggestions？

Reply:
My options are: 404, 503

Reply:
What about 200? Both Go and OpenResty use it.

Reply:
agree

Reply:
I will add this action to APISIX as well.

Reply:
update